uint16 command
uint8 result
uint8 progress
uint32 result_param2